elementary/monitor/api/tests/schema.py

```diff
@@ -50,6 +53,12 @@ class TestMetadataSchema(BaseModel):
     status: str
     test_created_at: Optional[str] = None
     days_diff: float
+    invocations_order: int
+    sample_data: Optional[Union[dict, List]] = None
```

Comment:
Why can this be both a dict and a list?

Reply:
Good question!
Currently we have a different samlpe_data structure for different test types (The UI is based on that).
This is something that we have from day 1, and we need to unify, but this isn't part of this PR 🙂